commodore/component_template.py

```diff
@@ -10,21 +11,27 @@
 from .helpers import yaml_load, yaml_dump
 
 
-def create_component(config, name, lib, pp):
+def create_component(config, name, lib, pp, owner, copyright):
+    slug = name.lower().replace(' ', '-')
```

Comment:
This is a duplication of the logic in `cookiecutter.json`. Can we reuse that somehow?

Reply:
I guess we could reuse the result of cookiecutter, but we'd have to do the cookiecutter stuff first (in a temp dir to be on the safe side), and then check if the component already exists in `dependencies`.

Reply:
Hmm, I like that idea.

Reply:
We need to come to an end here. I will create a ticket for that to be handled later.